src/auto-aspect-executor.ts

```diff
@@ -9,13 +9,14 @@ import { LazyDecorator } from './lazy-decorator';
  */
 @Injectable()
 export class AutoAspectExecutor implements OnModuleInit {
+  private readonly wrappedMethodCache = new WeakMap();
   constructor(
     private readonly discoveryService: DiscoveryService,
     private readonly metadataScanner: MetadataScanner,
     private readonly reflector: Reflector,
   ) {}
 
-  onModuleInit() {
+  async onModuleInit() {
```

Comment:
I think the async phrase is incorrect here

Reply:
I'll fix it 😅